app/controllers/course/experience_points_records_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+class Course::ExperiencePointsRecordsController < Course::ComponentController
+  load_resource :course_user, through: :course
+  load_and_authorize_resource :experience_points_record, through: :course_user,
+                                                         class: Course::ExperiencePointsRecord.name
+
+  def index # :nodoc:
+    @experience_points_records =
+      @experience_points_records.includes(creator: :course_users).references(:all)
```

Comment:
If I don't put this, bullet warns me of an N+1 query because I'm using `record.creator` in the "Awarded By" column.